app/services/decisions/income_decision_tree.rb

```diff
@@ -102,14 +106,21 @@ def after_employment_status
 
     def after_partner_employment_status
       if not_working?(form_object.partner_employment_status)
-        # TODO: route to partner income payments when route available
-        edit(:manage_without_income)
+        edit(:partner_income_payments)
```

Comment:
This needs to be `edit('/steps/income/partner/income_payments')`

But also, are we definitely meant to have it namespaced like this? I was expecting the route to be `income/income_payments_partner` to match `income/income_payments_client`